CHANGELOG.md

```diff
@@ -1,8 +1,48 @@
 # Database Tools Changelog
 
+## 100.5.4
+
+_Released 2022-07-19_
+
+We are pleased to announce version 100.5.4 of the MongoDB Database Tools.
+
+This release mostly consists of build failure fixes, support for new platforms, and tests against server version 6. The new platforms are Debian 11 on x86, Ubuntu 22.04 on x86 and ARM, and RHEL 3.9 on S390x. The version of Go driver used by the tools has been updated to 1.10.0.
```

Comment:
We should always refer to the major release as "6.0" for consistency.

Reply:
Also, you said RHEL 3.9. Should be 8.3.

Reply:
Oh also, I think the `s` is usually lower case in s390x.

Reply:
Fixed.